pom.xml

```diff
@@ -5,7 +5,7 @@
 
 	<artifactId>opensrp-server-core</artifactId>
 	<packaging>jar</packaging>
-	<version>2.3.2-SNAPSHOT</version>
+	<version>2.3.3-SNAPSHOT</version>
```

Comment:
This should be version `2.4.0-SNAPSHOT` as a lot of new functionality is being introduced

Reply:
Updated.